src/upload/storage.js

```diff
@@ -1,69 +1,150 @@
 import fs from 'fs';
 import mime from 'mime';
 import path from 'path';
+import { format } from 'util';
 import promisify from '../utils/promisify';
 
 const readFile  = promisify(fs.readFile);
 const stat      = promisify(fs.stat);
+const readDir   = promisify(fs.readdir);
+const makeDir   = promisify(fs.mkdir);
 const writeFile = promisify(fs.writeFile);
+const exists    = fsPath => stat(fsPath).then(() => true, () => false);
 
 export default class UploadStorage {
     constructor (uploadsRoot) {
         this.uploadsRoot = uploadsRoot;
     }
 
-    static async _loadFile (filePath) {
-        const fileContent = await readFile(filePath);
-        const stats       = await stat(filePath);
+    static async _getFilesToCopy (files) {
+        const filesToCopy = [];
+        const errs        = [];
 
-        return {
-            data: fileContent.toString('base64'),
-            info: {
-                lastModifiedDate: stats.mtime,
-                name:             path.basename(filePath),
-                type:             mime.lookup(filePath)
+        for (const file of files) {
+            try {
+                const stats = await stat(file.path);
+
+                if (stats.isFile())
```

Comment:
As I understand we don't have folders in the `Uploads root` folder. So, this condition is unnecessary.

Reply:
as far as I understand these files are not from `Uploads` folder